presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperatorFactory.java

```diff
@@ -123,6 +125,7 @@ private LookupJoinOperatorFactory(LookupJoinOperatorFactory other)
         referenceCount = other.referenceCount;
         outerOperatorFactory = other.outerOperatorFactory;
         probeHashGenerator = other.probeHashGenerator;
+        sharedPartitionedMemoryContext = other.sharedPartitionedMemoryContext;
```

Comment:
I think it should be duplicated, but @pnowojski should have a look as well. The relevant line is `LocalExecutionPlanner:435`